### PR TITLE
[vcpkg] Call vcpkg_copy_pdbs in each vcpkg install functions - script only

### PR DESCRIPTION
**Pull request by JackBoosY**
Script part of https://github.com/microsoft/vcpkg/pull/33330/

### CODE REVIEW

**Comment by JackBoosY** [comment]
@Cheney-W Please review this.

**Review by Cheney-W — APPROVED** [review]

**Review by BillyONeal — DISMISSED** [review]

**Comment by BillyONeal** [comment]
Quoth @ras0219-msft

>This was done as a separate step in case someone wanted to modify the results before calling `vcpkg_copy_pdbs`. For instance, removing a debug `.exe` before calling `vcpkg_copy_pdbs`.

=====================

Adding a "don't do `vcpkg_copy_pdbs` option" would be morally the same as the status quo without this change, so I'm opposed to doing that.

Perhaps we should do a post-build check of some sort instead?

File: scripts/cmake/vcpkg_install_make.cmake [hunk 1]
```diff
@@ -4,4 +4,6 @@ function(vcpkg_install_make)
         LOGFILE_ROOT
         ENABLE_INSTALL
     )
+
+    vcpkg_copy_pdbs()
```
Comment (BillyONeal) [inline scripts/cmake/vcpkg_install_make.cmake:8]: I don't think we should be touching things in `scripts/cmake` like this. I see you renamed it `- script only` which I thought meant you weren't leaving these touched but I see the edits still here?

**Comment by JackBoosY** [comment]
> Quoth @ras0219-msft
> 
> > This was done as a separate step in case someone wanted to modify the results before calling `vcpkg_copy_pdbs`. For instance, removing a debug `.exe` before calling `vcpkg_copy_pdbs`.
> 
> =====================
> 
> Adding a "don't do `vcpkg_copy_pdbs` option" would be morally the same as the status quo without this change, so I'm opposed to doing that.
> 
> Perhaps we should do a post-build check of some sort instead?

This scenario is rare. I think we can add an option `DISABLE_INSTALL_PDBS` that needs to be explicitly declared to each `vcpkg_*_build`/`vcpkg_*_install` to achieve this goal.
What do you think about this?

**Comment by JackBoosY** [comment]
Close this PR since it has conflict with the current solution.

**Pull request closed without merging**